### ISSUE TITLE
Transfer to live agent.

### ISSUE THREAD

**Issue by raoulduivestein** [davehorton/drachtio-dialogflow-phone-gateway#7]
Is it possible to transfer the call to a other live agent when a specific intent is triggert?

**Comment by davehorton** [comment]
Yes, have a look at the simwood branch.  Here is where I look at the returned intent to see if a call transfer is requested:

https://github.com/davehorton/drachtio-dialogflow-phone-gateway/blob/c088a99a134b838ee1c6c3c92a92503a3ced9926/lib/models/intent.js#L55

You can do something like that, modify the conditions to your needs.  Return the agent number to transfer to in a custom payload
